proteus/MeshTools.py

```diff
@@ -6137,6 +6137,7 @@ def getMeshIntersections(mesh, toPolyhedron, endpoints):
             intersections.update(((tuple(elementIntersections[0]), tuple(elementIntersections[1])),),)
     return intersections
 
+
```

Comment:
@Giovanni-Cozzuto-1989 this is not a big deal, but you could delete these blank line additions to the source.